config/samples/core_v1beta1_openstackcontrolplane_network_isolation.yaml

```diff
@@ -195,13 +280,25 @@ spec:
       serviceUser: ceilometer
   swift:
     enabled: true
+    proxyOverride:
+      route: {}
     template:
       swiftRing:
         ringReplicas: 1
       swiftStorage:
         replicas: 1
       swiftProxy:
         replicas: 1
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
```

Comment:
It seem swift has admin endpoint as well. I'm wondering if that needs any special handling:
```
$ openstack endpoint list
+----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------------------+
| ID                               | Region    | Service Name | Service Type | Enabled | Interface | URL                                                                  |
+----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------------------+
| 11e45a795de3451e98c3a2bf62cd8809 | regionOne | keystone     | identity     | True    | public    | http://keystone-public-openstack.apps-crc.testing                    |
| 20c6a4e3686a479789e7870996537d13 | regionOne | glance       | image        | True    | public    | http://glance-public-openstack.apps-crc.testing                      |
| 3d0e456e4865469f9f3621e36c16eb6c | regionOne | nova         | compute      | True    | internal  | http://nova-internal.openstack.svc:8774/v2.1                         |
| 4374ef4900b044f69be00230c83ff65d | regionOne | keystone     | identity     | True    | internal  | http://keystone-internal.openstack.svc:5000                          |
| 45f03dfc2b85471c983263d775acdbb4 | regionOne | placement    | placement    | True    | internal  | http://placement-internal.openstack.svc:8778                         |
| 4eef2629de4c45819be31b22a2ef0b3f | regionOne | nova         | compute      | True    | public    | http://nova-public-openstack.apps-crc.testing/v2.1                   |
| 59056b058f314a7ebac8f7f92ac233ac | regionOne | cinderv3     | volumev3     | True    | public    | http://cinder-public-openstack.apps-crc.testing/v3                   |
| 6d0cc311825a4c0a9d7137f386c7ef10 | regionOne | neutron      | network      | True    | internal  | http://neutron-internal.openstack.svc:9696                           |
| 72f3b740ddf24ea7bc4d1e7687960c47 | regionOne | glance       | image        | True    | internal  | http://glance-internal.openstack.svc:9292                            |
| 82dceca20c9d49328a89479299e69aca | regionOne | placement    | placement    | True    | public    | http://placement-public-openstack.apps-crc.testing                   |
| a9c701822ba54fb7ad69b9676f1f1367 | regionOne | swift        | object-store | True    | public    | http://swift-public-openstack.apps-crc.testing/v1/AUTH_%(tenant_id)s |
| c4a60cb5d49e48e38814e67039d28446 | regionOne | swift        | object-store | True    | internal  | http://swift-internal.openstack.svc:8080/v1/AUTH_%(tenant_id)s       |
| d0a3073a68cf4bd0990ad19609b23e17 | regionOne | neutron      | network      | True    | public    | http://neutron-public-openstack.apps-crc.testing                     |
| dadc2ab841cf407eb4406345f5212ca5 | regionOne | cinderv3     | volumev3     | True    | internal  | http://cinder-internal.openstack.svc:8776/v3                         |
| db0760949d6d49d2b904d368514b09cb | regionOne | swift        | object-store | True    | admin     | http://swift-admin.openstack.svc:8080                                |
+----------------------------------+-----------+--------------+--------------+---------+-----------+----------------------------------------------------------------------+
```

Reply:
just point out again that the admin endpoint is not deprecated or unsupported in OpenStack.

keystone somiple removed the need fo using it in keystone they did not remove the feature.
the internal endpoint is meant to be accsessabel to tenant workloads especially for the swirft and glance services. 

the special version of glance that has the multiple location info should be exposed via the admin endpoint not the internal one. i have given that feedback beferore prior to implementing the current security mitigation.

the admin endpoint should not need routes. they should not generally be accessible publically.

Reply:
yes, I remember we discussed that on slack a while back. we can re-add the admin endpoint to the services if that is required.